src/helm/benchmark/window_services/anthropic_window_service.py

```diff
@@ -23,3 +28,7 @@ def max_request_length(self) -> int:
         Anthropic does not include the start of sequence token.
         """
         return self.max_sequence_length
+
+    @property
+    def tokenizer_name(self) -> str:
```

Comment:
I think you need two separate classes: `AnthropicWindowService` and `LegacyAnthropicWindowService`

Then these changes go in just `AnthropicWindowService`. And in `window_service_factory` [here](https://github.com/stanford-crfm/helm/blob/80ecb204a9fed6a54a53e1f5950e9c1e145acddc/src/helm/benchmark/window_services/window_service_factory.py#L86-L87), you would instantiate the legacy window service for the legacy model.